ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/simulation/sensors/RDXSimulatedSensorFactory.java

```diff
@@ -247,4 +247,31 @@ public static RDXHighLevelDepthSensorSimulator createChestRightBlackflyForObject
       highLevelDepthSensorSimulator.setupForROS2Color(pubSubImplementation, PerceptionAPI.BLACKFLY_VIDEO.get(RobotSide.RIGHT));
       return highLevelDepthSensorSimulator;
    }
+
+   public static RDXHighLevelDepthSensorSimulator createChestZED2ForObjectDetection(HumanoidReferenceFrames referenceFrames,
```

Comment:
Is it ZED2 or blackfly right? Below there is `"Blackfly Right for Object Detection"`